contrib/nydus-overlayfs/cmd/main.go

```diff
@@ -0,0 +1,144 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"syscall"
+
+	"github.com/pkg/errors"
+	"github.com/urfave/cli/v2"
+	"golang.org/x/sys/unix"
+)
+
+const (
+	nydusOptionKey = "nydusoption"
+)
+
+var (
+	Version   = "development"
+	BuildTime = "unknown"
+)
+
+/*
+   containerd run fuse.mount format: nydus-overlayfs overlay /tmp/ctd-volume107067851
+   -o lowerdir=/foo/lower2:/foo/lower1,upperdir=/foo/upper,workdir=/foo/work,nydusoption={...},dev,suid]
+*/
+type mountArgs struct {
+	fsType  string
+	target  string
+	options []string
+}
+
+func parseArgs(args []string) (*mountArgs, error) {
+	margs := &mountArgs{
+		fsType: args[0],
+		target: args[1],
+	}
+	if margs.fsType != "overlay" {
+		return nil, errors.New("fsType only support overlay")
+	}
+	if len(margs.target) == 0 {
+		return nil, errors.New("target can not be empty")
+	}
+	if args[2] == "-o" && len(args[3]) != 0 {
+		for _, opt := range strings.Split(args[3], ",") {
```

Comment:
Just a nitt.
I am curious if we can use package like `"github.com/urfave/cli/v2"` to extract mount point and mount options out of command line arguments since it is a generic step